### DISCUSSION TITLE
Do not ask for confirmation when doing coc-rename?

### DISCUSSION THREAD

**Discussion by avalonv** [neoclide/coc.nvim#4279] [Q&A] [answered]
![Peek 2022-10-11 12-45](https://user-images.githubusercontent.com/29720696/195138878-cfae257c-67b3-4216-a21d-2fac6400f8d0.gif)
 
On previous versions, when I did `coc-rename` using the default keybinding (`<leader>rn`) it would change the symbol under the cursor and show the " Use ':wa' to save changes or ':CocCommand workspace.inspectEdit' to inspect."  message without requiring further input. But now it also asks for confirmation (see above). How can I revert to the old behaviour?

**Comment by chemzqm** [comment, marked as answer]
It should not show that message when rename happens on current buffer only.
You can use `"coc.preferences.enableMessageDialog": true` to use dialog window.
